its/Models/DotnetFrameworkTestCase.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
+ * Azure DevOps extension for SonarQube/SonarCloud ITs
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto: info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+namespace IntegrationTests.Models
+{
+	public class DotnetFrameworkTestCase : BuildTestCase
+	{
+		public DotnetFrameworkTestCase()
+		{
+			base.Coverage = Double.Parse(EnvironmentVariableWrapper.GetVariableOrThrow("ITS_DOTNET_EXPECTED_COVERAGE"));
+			base.NcLocs = Int64.Parse(EnvironmentVariableWrapper.GetVariableOrThrow("ITS_DOTNET_EXPECTED_NCLOCS"));
+			base.PipelineName = EnvironmentVariableWrapper.GetVariableOrThrow("ITS_DOTNET_PIPELINE_NAME");
+			base.ProjectKey = EnvironmentVariableWrapper.GetVariableOrThrow("ITS_DOTNET_SC_PROJECT_KEY");
+			base.LogPrefix = "S4NET - .NET Framework";
+		}
```

Comment:
You can save a ton of duplications if you move the initialization to the base class and provide just the environment variable infix.

It will also enforce regular structure for the environment variable names.
```suggestion
		public DotnetFrameworkTestCase() : base("DOTNET", "S4NET - .NET Framework") {}
```
For sake of simplicity, we can use the infix as the LogPrefix as well.
```suggestion
		public DotnetFrameworkTestCase() : base("DOTNET") {}
```

Reply:
Having that, we can remove all 4 of these classes and initialize the value only in the tests.